Makefile

```diff
@@ -19,7 +19,7 @@ example/cmd/device-simple/device-simple:
 
 docker:
 	docker build \
-		-f docker/Dockerfile \
+		-f example/cmd/device-simple/Dockerfile \
 		--label "git_sha=$(GIT_SHA)" \
 		-t edgexfoundry/docker-device-sdk-simple:$(GIT_SHA) \
 		-t edgexfoundry/docker-device-sdk-simple:$(VERSION)-dev \
```

Comment:
This works for me only after manually adding these two lines:
```
		--build-arg http_proxy=${HTTP_PROXY} \
		--build-arg https_proxy=${HTTPS_PROXY} \
```
These won't hurt folks that have no need to set proxy info, but makes it easier for those of us who do.

Reply:
Yes, I think that would be a good fix for our proxy troubles.